docs/deploy.md

```diff
@@ -1,108 +1,75 @@
 ## deploy task
-
-The `deploy` task supports deployment of one or more applications to the Liberty server.
+---
+The `deploy` task copies applications generated by the Gradle project to the Liberty server's `dropins` or `apps` directory. If an application is deployed while the server is running the task will verify that the application has started successfully.
 
 ### dependsOn
-`deploy` depends on `libertyStart` for a running server.
-
-### Properties
-
-See the [Liberty server configuration](libertyExtensions.md#liberty-server-configuration) properties for common server configuration.
+`deploy` depends on any tasks of type `war` or `ear` so the package is ready before installation.  
+`deploy` also depends on `libertyCreate` to ensure that the server exists.
 
-The `deploy` task uses a `deploy` block inside the `liberty` block to define task specific behavior.
+### Parameters
 
-| Attribute | Type | Since | Description | Required |
-| --------- | ---- | ----- | ----------- | ---------|
-| file| String | 1.0 | Location of a single application to be deployed. The application type can be war, ear, rar, eba, zip, or jar. | Yes, only when a single file will be deployed. |
-| dir| String | 1.0 | Location of the directory where are the applications to be deployed.| Yes, only when multiples files will be deployed and `file` is not specified.|
-| include| String | 1.0 | Comma or space-separated list of patterns of files that must be included. All files are included when this attribute is omitted.| No |
-| exclude| String | 1.0 | Comma or space-separated list of patterns of files that must be excluded. No files are excluded when this attribute is omitted.| No |
+The `deploy` task can be configured using the `deploy` block. Below are the properties that can be set in the `deploy` block.
```

Comment:
Shouldn't we specify that the `deploy` block is inside the `server` block?

Reply:
Agreed.